drivers/storage/class/ramdisk/ramdisk.inf

```diff
@@ -0,0 +1,69 @@
+; RAMDISK.INF
+
+; Installation of RAMDISK DRIVES
+
+[Version]
+Signature  = "$Windows NT$"
+LayoutFile = layout.inf
+Class      = Ramdisk
+ClassGUID  = {9D6D66A6-0B0C-4563-9077-A0E9A7955AE4}
+Provider   = %ReactOS%
+DriverVer  = 06/29/2008,1.00
+
+[DestinationDirs]
+RamdiskBus_CopyFiles.NT = 12
+
+[ClassInstall32.NT]
+AddReg = RamdiskClass.NT.AddReg
+
+[RamdiskClass.NT.AddReg]
+HKR, , ,               0, %RamdiskClassName%
+HKR, , Icon,           0, "-5"
+
+[Manufacturer]
+%ReactOS% = RamdiskDevice
+
+[RamdiskDevice]
+%RamdiskBus.Desc% = RamdiskBus_Inst, Ramdisk, DetectedInternal\Ramdisk, Detected\Ramdisk
+%RamdiskVolume.Desc% = RamdiskVolume_Inst, Ramdisk\RamVolume
+
+;----------------------------- RAMDISK BUS -----------------------------
+
+[RamdiskBus_Inst.NT]
+CopyFiles = RamdiskBus_CopyFiles.NT
+
+[RamdiskBus_Inst.NT.Services]
+AddService = ramdisk, 2, RamdiskBus_Service_Inst
+
+[RamdiskBus_Service_Inst]
+DisplayName    = %Ramdisk.ServiceDesc%
+ServiceType    = 1
+StartType      = 0
+ErrorControl   = 1
+ServiceBinary  = %12%\ramdisk.sys
+
+[RamdiskBus_CopyFiles.NT]
+ramdisk.sys
+
+;----------------------------- RAMDISK VOLUME -----------------------------
+
+[RamdiskVolume_Inst.NT]
+
+[RamdiskVolume_Inst.NT.Services]
+AddService = , 2
+
+;-------------------------------- STRINGS -------------------------------
+
+[Strings]
+ReactOS = "ReactOS Team"
+RamdiskClassName = "Ramdisk"
+RamdiskBus.Desc = "ReactOS RAM Disk Controller"
+RamdiskVolume.Desc = "ReactOS RAM Disk Device (volume)"
```

Comment:
The controller and volume devices are not ReactOS-specific, so better name them like that:
```suggestion
RamdiskBus.Desc = "RAM disk controller"
RamdiskVolume.Desc = "RAM disk volume"
```
(lowercase names for consistency with other standard device names)

Reply:
On Windows:
![image](https://user-images.githubusercontent.com/1969829/91301179-ca207900-e7a4-11ea-96c7-fccd54aefe73.png)
(Translation:
"Pilote" == "Driver"
"Contrôleur de disque virtuel Windows" == "Windows Virtual Disk Controller"
"Périphérique de disque virtuel Windows (volume)" == "Windows Virtual Disk Peripheral (volume)"
)

Reply:
Ok then a second suggestion :smile: Feel free to choose between my first one and this one
```suggestion
RamdiskBus.Desc = "RAM disk controller"
RamdiskVolume.Desc = "RAM disk device (volume)"
```

Reply:
![image](https://user-images.githubusercontent.com/578406/91301741-16bc8200-e7ae-11ea-9747-72b974aa1657.png)